dlp/deIdentifyWithExceptionList.js

```diff
@@ -0,0 +1,132 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//  title: Deidentify with Exception List
```

Comment:
Please use De-identify when using in an English sentence. Here and other lines added by you.

Reply:
@annadata https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3099 on this PR, I used the one that you mentioned, but I got a comment to change the case of the first letter. So shall i go with this one or the other one?